app/core/Tracker2/DriftTracker.js

```diff
@@ -121,7 +121,13 @@ export default class DriftTracker extends BaseTracker {
       enableWelcomeMessage: chatEnabled,
       enableCampaigns: chatEnabled,
       enableChatTargeting: chatEnabled,
-    })
+    });
+
+    if (chatEnabled) {
+      this.driftApi.widget.show();
```

Comment:
I don't believe we need the `show`, especially in case the user has already closed the chat bubble, but I could be wrong. Probably fine to ship with or without, and adjust as we get more feedback.

Reply:
Agreed. Has pros and cons. Will update it on based of feedback.